roles/cobbler/templates/kickstarts/cephlab_ubuntu.preseed

```diff
@@ -3,6 +3,12 @@
 # Fetch the os_version from the distro using this profile.
 #set os_version = $getVar('os_version','')
 
+# Fetch Ubuntu version (e.g., 14.04)
+#set distro = $getVar('distro','').split("-")[1]
+
```

Comment:
Is the variable namespace where getVar('distro') is defined different from the namespace where set defines it (i.e. are we redefining distro)?  If so, maybe we should choose a different local name in case something else depends on the current value